lighthouse-core/report/v2/report-styles.css

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ :root {
+  --text-font-family: '.SFNSDisplay-Regular', 'Helvetica Neue', 'Lucida Grande', sans-serif;
+  --body-font-size: 13px;
+
+  --secondary-text-color: #565656;
+  /*--accent-color: #3879d9;*/
+  --poor-color: #df332f;
+  --good-color: #2b882f;
+  --average-color: #ef6c00; /* md orange 800 */
+  --warning-color: #757575; /* md grey 600 */
+
+  --report-border-color: #ebebeb;
+
+  --lh-score-width: 50px;
+  --lh-score-margin: 20px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: var(--text-font-family);
+  font-size: var(--body-font-size);
+  margin: 0;
+  line-height: var(--body-line-height);
+}
+
+.--hidden {
+  display: none !important;
+}
+
+/* Text */
+
+/*.lighthouse-text {
+  white-space: nowrap;
```

Comment:
These aren't needed yet so I commented them out. There was also a lot of unnecessary CSS flexbox that was removed. Divs are already block level elements 👍